frontend/src/concepts/pipelines/content/createRun/types.ts

```diff
@@ -43,6 +43,7 @@ export type RunParam = {
 export type RunFormData = {
   project: ProjectKind;
   nameDesc: { name: string; description: string };
+  pipelineLoaded: boolean;
```

Comment:
nit: this should be called `pipelinesLoaded` since we are waiting on loading all the pipelines.